tools/serverpod_cli/lib/src/analyzer/dart/endpoints_analyzer.dart

```diff
@@ -70,22 +86,32 @@ class EndpointsAnalyzer {
     for (var context in collection.contexts) {
       var analyzedFiles = context.contextRoot.analyzedFiles();
       for (var filePath in analyzedFiles) {
-        var errors = await context.currentSession.getErrors(filePath);
-        if (errors is ErrorsResult) {
-          for (var error in errors.errors) {
-            if (error.severity == Severity.error) {
-              // TODO: Figure out how to include line number
-              errorMessages.add(
-                '${error.problemMessage.filePath} Error: ${error.message}',
-              );
-            }
-          }
-        }
+        var fileErrors =
+            await _getErrorsForFile(context.currentSession, filePath);
+        errorMessages.addAll(fileErrors);
       }
     }
     return errorMessages;
```

Comment:
Question: Is this method needed at all?

Reply:
No, not really. We only utilize it tests at the moment but with the change in this PR we get information about Dart syntax errors anyway.

Will remove it 👍